docs/contents/components/forms/year-picker/index.ja.mdx

```diff
@@ -247,9 +297,16 @@ import "dayjs/locale/ja"
 
 ```tsx
 <VStack>
-  <YearPicker isInvalid variant="outline" placeholder="outline" />
-  <YearPicker isInvalid variant="filled" placeholder="filled" />
-  <YearPicker isInvalid variant="flushed" placeholder="flushed" />
+  <For each={["outline", "filled", "flushed"]}>
```

Comment:
@kaehehehe 
It seems that using `<For />` has actually reduced readability. Why don't we try replacing `<For />` with something else?

Reply:
I think it's better to use the For component. Why do you think the readability is lower?